tile-service/src/main/java/com/oculusinfo/tile/rest/config/ConfigServiceImpl.java

```diff
@@ -88,46 +81,15 @@ public File findResourceConfig(String name) throws URISyntaxException {
     }
 
     protected Map<String, String> buildReplacements() throws ConfigException {
-        String pathToProperties = getPathToProperties();
-        if (StringUtils.isEmpty(pathToProperties)) {
-            return null;
-        }
-
-        try (InputStream input = new FileInputStream(pathToProperties)) {
-            Map<String, String> replacements = new HashMap<>();
-            Properties properties = new Properties();
-            properties.load(input);
-            Enumeration e = properties.propertyNames();
-            while (e.hasMoreElements()) {
-                String key = (String) e.nextElement();
-                replacements.put(key, properties.getProperty(key));
-            }
-            return replacements;
-        } catch (IOException e) {
-            throw new ConfigException(String.format("Unable to read properties file %s", pathToProperties), e);
-        }
-    }
-
-    protected String getPathToProperties() throws ConfigException {
-        String pathToProperties;
-        String configEnvVar = System.getenv(CONFIG_ENV_VAR);
-        if (StringUtils.isNotEmpty(configEnvVar)) {
-            pathToProperties = configEnvVar;
-            LOGGER.warn("{} environment variable is set, using properties file {} ", CONFIG_ENV_VAR, pathToProperties);
-        } else {
-            pathToProperties = getPathToPropertiesFromClasspath();
-            LOGGER.warn("{} environment variable NOT set, using default file from classpath {}", CONFIG_ENV_VAR, pathToProperties);
-        }
-        return pathToProperties;
-    }
-
-    protected String getPathToPropertiesFromClasspath() {
-        String path = null;
-        URL resource = this.getClass().getClassLoader().getResource(DEFAULT_CONFIG_PROPERTIES);
-        if (resource != null) {
-            path = resource.getPath();
+    	ConfigPropertiesUtil configUtil = new ConfigPropertiesUtil();
```

Comment:
I believe the pattern is to use Guice for dependency injection, rather than invoke new in a method.